packages/core/src/commands/urls.ts

```diff
@@ -12,7 +16,11 @@ export const urls = async ({ envId, rootUrl, clientId, projectName, serviceAndPo
 
   const { tunnels } = await queryTunnels({ tunnelUrlForService, retryOpts: { retries: 2 } })
 
+  const credentials = includeAccessCredentials ? await getUserCredentials(jwtGenerator(tunnelingKey)) : undefined
   return flattenTunnels(tunnels)
+    .map(x => ({ ...x,
+      url: credentials
```

Comment:
this is a different version of the logic at up/index.ts

Reply:
Fixed